test/dotnet-msbuild.Tests/GivenMsbuildForwardingApp.cs

```diff
@@ -71,7 +72,7 @@ public void ItSetsOrIgnoresTelemetrySessionId()
             var envVar = "DOTNET_CLI_TELEMETRY_SESSIONID";
             var startInfo = new MSBuildForwardingApp(new string[0], msbuildPath)
                 .GetProcessStartInfo();
-            (startInfo.Environment[envVar] == null || int.TryParse(startInfo.Environment[envVar], out _))
+            (startInfo.Environment[envVar] == null || Guid.TryParse(startInfo.Environment[envVar], out _))
```

Comment:
>Guid [](start = 54, length = 4)

How was this only caught by arcade move?

Reply:
Right. I saw it fail on a couple of runs in the past but never spent the time tracking it down. This time I did :Smile: